src/components/NavBar/NavBar.tsx

```diff
@@ -55,10 +58,11 @@ function previewTheme(theme) {
 function exitThemePreview() {
     _update_theme(data.get("theme"));
 }
+
 function setTheme(theme) {
-    data.set("theme", theme);
-    _update_theme(theme);
+    data.set("theme", theme, data.Replication.REMOTE_OVERWRITES_LOCAL); // causes _update_theme to be called via the data.watch() in constructor
```

Comment:
I'm thinking this should be LOCAL_OVERWRITES_REMOTE so that if you have multiple devices, if you set it manually on any particular device it'll stay stuck for that device?

Note that this will make more sense as time goes on and you begin to use new devices or browser local storage gets cleared for whatever reason.. right now, all devices will have the local value for theme set so it'll be a bit harder to notice this, but I think in time `LOCAL_OVERWRITES_REMOTE` is the right way to go.

Come to think of it, that name is a misnomer, it should be `LOCAL_OVERRIDES_REMOTE` or something, it doesn't do any writing..

Reply:
Now we are coming to "what people would want" instead of "make it work", at last 😄 

I had it set with REMOTE_OVERWRITES_LOCAL so that I definitely saw the changes every time, as much as for any other reason (also this was the equivalent of how it worked before those new funky options were added).

Overall, it seems like your logic is correct - if it's the case that when I log into a new device, I get the remote setting automatically (should be true, needs to be confirmed) then you would argue that if I change that locally, then I want it different here.

BUT what if I just changed my mind about what I want everywhere?

Reply:
... I feel more inclined to release it with it always changing everywhere.   When someone comes up with "hey I want liht mode on my phone" we could look at this problem at that point.  In the mean time, everyone would experience cool synching  😁

Reply:
But if it's `REMOTE_OVERWRITES_LOCAL`, you *can't* have separate themes for different devices, pretty sure that's going to annoy some folks.

I'm pondering if there's a better way to approach this rather than having remote actually stomp on the local values too.. might noodle on it a little bit more here today.

I'll get this merged and we can feel it out a bit on beta before we release.